dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/fat/AcmeValidityAndRenewTest.java

```diff
@@ -119,6 +133,7 @@ public void serverBelowMinRenew() throws Exception {
 
 			assertNotNull("Should log warning on minimum renewBeforeExpiration",
 					server.waitForStringInLog("CWPKI2051W"));
+			AcmeFatUtils.assertAndGetServerCertificate(server, caContainer);
 
 		} finally {
 			Log.info(this.getClass(), methodName, "TEST 1: Shutdown.");
```

Comment:
methodName could be replaced by testName.getMethodName()

Reply:
Done